README.md

```diff
@@ -1,24 +1,43 @@
-# README
+# Hello rails + react
+An app to test how to create a rails + react application
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## Built With:
 
-Things you may want to cover:
+- Ruby & Rails
+- React & Redux
 
-* Ruby version
+## Getting Started
+- To get a local copy of this project clone this repository or download the Zip folder:
+git clone https://github.com/Gopxfs/hello-rails-react.git
 
-* System dependencies
+- To get started, In the root directory run:
+gem install bundler
 
-* Configuration
+- Then run:
+bundle install
 
-* Database creation
+- Run ruby app in the Command Line:
+rails s
 
-* Database initialization
```

Comment:
* Kindly add more setup instructions to your readme file so that visitors of your repository install and use your app easily